num2words/lang_RO.py

```diff
@@ -0,0 +1,126 @@
+import num2words
+import num2words.lang_EU
+
+class Num2Word_RO(num2words.lang_EU.Num2Word_EU):
+    def set_high_numwords(self, high):
+        max = 3 + 3*len(high)
+        for word, n in zip(high, list(range(max, 3, -3))):
+            self.cards[10**n] = word + "ilion"
+
+    def setup(self):
+        self.negword = "minus "
+        self.pointword = "punct"
+        self.errmsg_nornum = "Only numbers may be converted to words."
+        self.exclude_title = ["si", "punct", "minus"]
+
+        self.mid_numwords = [(1000000, "milio/n/ane"), (1000, "mi/e/i"), (100, "sut/a/e"),
+                             (90, "nouazeci"), (80, "optzeci"), (70, "saptezeci"),
+                             (60, "saizeci"), (50, "cincizeci"), (40, "patruzeci"),
+                             (30, "treizeci"), (20, "douazeci")]
+        self.low_numwords = ["douazeci", "nouasprezece", "optisprezece", "saptisprezece",
+                             "saisprezece", "cincisprezece", "paisprezece", "treisprezece",
+                             "doisprezece", "unsprezece", "zece", "noua", "opt",
+                             "sapte", "sase", "cinci", "patru", "trei", "doi",
+                             "unu", "zero"]
+        self.ords = { "unu"    : "primul",
+                      "doi"    : "al doilea",
+                      "trei"   : "al treilea",
+                      "cinci"  : "al patrulea",
+                      "opt"  : "al optulea",
+                      "noua"   : "al noualea",
+                      "doisprezece" : "al doisprezecelea" }
+        self._ro_feminine = { "unu": "una", "doi": "doua" }
+        self._ro_short = {'unu': 'un', 'una': 'o'}
+        self._ro_should_feminine = [1000, 100]
+
+    def merge(self, xxx_todo_changeme, xxx_todo_changeme1):

```

Comment:
These were actually copied from the lang_EN.py from the pypi published version, but I will address these (and the other two problems) and get back to you. Thanks!